courses/models_test.py

```diff
@@ -15,13 +16,56 @@
     ProgramEnrollmentFactory,
     ProgramFactory,
 )
-from courses.models import CourseRunEnrollment
+from courses.models import Course, CourseRunEnrollment, ProgramRequirement
 from main.test_utils import format_as_iso8601
 from users.factories import UserFactory
 
 pytestmark = [pytest.mark.django_db]
 
 
+@pytest.fixture
+def program_with_requirements():
+    program = ProgramFactory.create()
+    required_courses = CourseFactory.create_batch(3, program=program)
+    elective_courses = CourseFactory.create_batch(3, program=program)
```

Comment:
Probably not important, but DEDP currently has 4 electives:

14.009x
14.73x
14.740x
14.750x